src/github/nag.rs

```diff
@@ -870,8 +874,26 @@ impl<'a> RfcBotCommand<'a> {
                         diesel::insert(&new_concern)
                             .into(fcp_concern)
                             .execute(conn)?;
+            
+                        // Take us out of FCP and back into PFCP if need be:
+                        if proposal.fcp_start.is_some() {
+                            // Update DB: FCP is not started anymore.
+                            proposal.fcp_start = None;
+                            match diesel::update(fcp_proposal.find(proposal.id))
```

Comment:
Do you think the bot should comment when it's doing this? Could be confusing to do this without any visible status update?

cc @aturon

Reply:
My thinking is that:
```rust
let _ = issue.add_label(Label::PFCP);
issue.remove_label(Label::FCP);
```
serves as visible status update that the bot reacted to the concern :)

Reply:
Yeah I think that suffices, given that the concern registration is already a comment.